frontend/src/config.json

```diff
@@ -0,0 +1,127 @@
+{
+    "Theme": {
+        "Rainbow": {
```

Comment:
Minor detail but we maybe can change the name of the theme since it is not rainbow anymore